float64      rating
int64        date_uploaded_unix
YtsTorrent[] torrents
int64        year
int64        id
string       large_cover_image
string       title
string       title_long
string       state
string       small_cover_image
YtsState     hs_state
string       mpa_rating
string       background_image_original
string       background_image
string       title_english
string       imdb_code
string       medium_cover_image
string       slug
string       description_full
string       language
string       url
string       summary
string       synopsis
string       date_uploaded
int64        runtime
string       yt_trailer_code
string[]     genres
